src/snyk/snykOss/services/ossService.ts

```diff
@@ -60,7 +59,7 @@ export class OssService extends CliService<OssResult> {
 
     this.viewManagerService.refreshOssView();
 
-    if (configuration.shouldAutoScanOss) {
+    if (this.config.shouldAutoScanOss) {
```

Comment:
@michelkaporin just want to clarify small TS convention question for myself. In some cases I see 

```
if (statement) {
   oneLineInIf()
}
```

In some cases see

`if (statement) oneLineInIf()`

Both options comply with the convention? Interesting because for Java convention only the first one.

Reply:
Yes, we allow both of them, otherwise you'd see ESLint warning within the code. JS is not that strict on braces rules, also allowing writing the following:
```js
 if (statement) oneLineIf
```

We don't enforce it too as it's a pure matter of taste 👅